webpack/components/SCCProductPage/components/SCCProductPicker/components/SCCTreePicker/components/SCCRepoPicker/index.js

```diff
@@ -38,7 +38,7 @@ const setRepoSelection = (
       res = sccRepos.filter(
         (repo) =>
           (!repo.name.includes('Debug') &&
-            !repo.name.includes('Source-Pool') &&
+            !repo.name.includes('-Source') &&
```

Comment:
does this detect both, old SLES15 Source-Pool and new SLES 16 -Source?

Reply:
Yes, the new ones have '-Source' as suffix and the old ones '-Source-Pool'